paymentrequest/android-pay/demo.js

```diff
@@ -5,15 +5,15 @@ function onBuyClicked() {
   var supportedInstruments = [{
     supportedMethods: ['https://android.com/pay'],
     data: {
-      environment: 'TEST',
-      merchantId: '123456',
+      merchantName: 'Android Pay Demo',
+      merchantId: '00184145120947117657',
```

Comment:
Added the appropriate comments and notes on abuse.

Merchant ID cannot be abused, because it's tied to the origin of the website. This merchant ID works only on googlechrome.github.io, rsolomakhin.github.io, and emerald-eon.appspot.com.

Stripe publishable key (pk_*) cannot be abused, because one needs the corresponding Stripe secret key (sk_*) to initiate a transaction on the server.